etl/steps/data/garden/emissions/2023-11-23/national_contributions.meta.yml

```diff
@@ -1,330 +1,323 @@
-all_sources:
-  - source_jones: &source-jones
-      name: Our World in Data based on Jones et al. (2023)
-      published_by: |
-        Our World in Data based on Jones, Matthew W., Peters, Glen P., Gasser, Thomas, Andrew, Robbie M., Schwingshackl, Clemens, Gütschow, Johannes, Houghton, Richard A., Friedlingstein, Pierre, Pongratz, Julia, & Le Quéré, Corinne. (2023). National contributions to climate change due to historical emissions of carbon dioxide, methane and nitrous oxide [Data set]. In Scientific Data (2023.1). Zenodo.
-      url: https://zenodo.org/record/7636699#.ZFCy4exBweZ
-      date_accessed: "2023-05-02"
-      publication_date: "2023-02-13"
+definitions:
```

Comment:
nit - I usually define decimal places to display as it's been something that I often then got feedback on during the review process. not sure how relevant it is for your purposes though

Reply:
I'll try to add consistency on decimal places when going through the all-charts block.